### ISSUE TITLE
Company name after author name looks odd

### ISSUE THREAD

**Issue by hlflanagan** [rfc-format/draft-iab-rfc-css-bis#18]
(based on sample2.html)
Perhaps on a separate line, or would this just be better comma separated.

Will need to consider how this would look in an RFC that has authors both with and without company affiliation. What would that do to the spacing?

**Comment by martinthomson** [comment]
I agree, I was going to open the same issue.  A comma, or even parentheses, would help distinguish the two.
